torchgeo/samplers/batch.py

```diff
@@ -73,6 +74,7 @@ def __init__(
         batch_size: int,
         length: int,
         roi: Optional[BoundingBox] = None,
+        sample_mode: int = SIZE_IN_CRS_UNITS,
```

Comment:
This needs to be documented in the docstring below with a list of possible values

Reply:
SIZE_IN_PIXELS may be a better default, I'm not worried about breaking backwards compatibility

Reply:
Also, `units` may be a better parameter name